Blog/tests.py

```diff
@@ -73,6 +74,12 @@ def test_changes_edits_the_post_successfully(self):
         fetch_updated_data = Blog.objects.get(title='test title to check the updation')
         self.assertEquals(fetch_updated_data.content, 'test content to check the updation is working' )
 
+    def test_rendering_newLine_as_brk(self):
+        response = self.client.post(reverse('Blog:homepage'), {'title': 'ANONYMOUS BLOG POST RENDERING NEW LINES AS BREAK', 'content': 'hai I am using Django for development,\n when I render it to a template should  show the newline character'}, follow=True)
+        self.assertContains(response, '<br />', status_code=200)
```

Comment:
Your template can also contain `<br>` so this test wont always work. So if you have `first line\nsecond line` ensure that `first line` and `second line` have a `<br />` between them. Any other `<br />` should not influence the result.